src/ext/h2c.js

```diff
@@ -2,15 +2,17 @@
  * Implements the methods of hashing to elliptic curves
  * that are described in draft-irtf-cfrg-hash-to-curve
  * @author Alex Davidson
- * TODO: make constant-time
+ * Note: None of the algorithms are written in a constant-time manner.
+ * Fortunately, in the use-case of Privacy Pass with Cloudflare this does not
+ * matter since the computation is masked by human completion of the CAPTCHA.
  */
 
  /* global sjcl */
  /* exported h2Curve */
 
-// compatible curves
-const p256 = sjcl.ecc.curves.c256;
-const FFp = p256.field;
+ // We only support hashing to curve using the affine representation for now
+ const SWU_MODE = 0;
```

Comment:
Here is good to indicate the meaning of `SWU_MODE = 0` and other possible options